src/items/Document.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import FileDocument from './FileDocument';
+
+const meta: Meta<typeof FileDocument> = {
+  title: 'Items/Document File',
```

Comment:
I think Document is actually what it is, but because of our other nomenclature I would prefer something a bit closer to "viewer" and to the library you used.

```suggestion
  title: 'Items/FileDocumentViewer',
```